Sources/Brave/Frontend/Browser/Search/BraveSearchManager.swift

```diff
@@ -177,7 +177,7 @@ class BraveSearchManager: NSObject {
     guard var components = URLComponents(string: fallbackProviderURLString) else { return }
 
     var queryItems: [URLQueryItem] = [
-      .init(name: "q", value: query.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed))
```

Comment:
Curious Where do we do other encoding ?

Reply:
seems to be done when you submit a query from the in-app url bar